content/docs/user-guide/packaging/project-export/project-export-android.md

```diff
@@ -8,15 +8,15 @@ weight: 410
 This guide covers how to use project export tooling to build and deploy games on Android. We will use the Android export script to generate a Gradle project, prepare assets, and then build the project. The end result of export should be an APK file that can be installed on your Android device, along with an additional step to auto-install on a connected phone.
 
 {{< note >}}
-To learn more about the project export tooling, please consult the page [Project Export CLI Tool.](/docs/user-guide/packaging/project-export/project-export-cli)
+To learn more about the project export tooling in general, please consult the page [Project Export for Windows and Linux.](/docs/user-guide/packaging/project-export/project-export-pc)
 {{< /note >}}
 {{< note >}}
 To learn more about how to manually export an Android project, please consult the page [Generating an Android Project](https://development--o3deorg.netlify.app/docs/user-guide/platforms/android/generating_android_project_windows/#prerequisites).
```

Comment:
Notice that there are multiple places still using `development--o3deorg.netlify.app` in links. Maybe we update all to relative paths?

For this one, it would be like this:
```suggestion
To learn more about how to manually export an Android project, please consult the page [Generating an Android Project](/docs/user-guide/packaging/project-export-android/#prerequisites).
```

But I am not sure if `#prerequisites` can work here.